### DISCUSSION TITLE
Prossimi sviluppi

### DISCUSSION THREAD

**Discussion by AndreFra96** [AndreFra96/strava-tools#15] [Ideas]
Utilizziamo questa discussione per parlare dei prossimi sviluppi e di quello che ci viene in mente che vorremmo inserire all'interno del progetto

**Comment by AndreFra96** [comment]
# Similarita GPX

Mi piacerebbe provare insieme a implementare un algoritmo di similarità fra tracce gps. 

Prendendo due attività diverse (ognuna con la propria traccia GPS che indica il percorso effettuato) l'obiettivo sarebbe quello di stabilire quanto i due percorsi hanno in comune; In questo modo potremmo poi proseguire confrontando le prestazioni fra attività simili. 

Questa funzionalità è presente in Strava ma non è molto chiaro in base a cosa venga stabilità la similarità e non è possibile estrarre questa informazione tramite le API.

@rafaelventura204 che ne pensi?

**Reply by rafaelventura204** [reply]
Ci sta, come idea non è male mi piace
